### DISCUSSION TITLE
Draw lines, rectangles

### DISCUSSION THREAD

**Discussion by MisterOzzy** [reactivemarkets/react-financial-charts#469] [General]
Is there possible draw lines, rectangles and so on from user perspective (do not draw from like 'code')? Do you have some examples?

**Comment by markmcdowell** [comment]
Yes you can, examples to be added this week. hopefully!

https://github.com/reactivemarkets/react-financial-charts/tree/master/packages/interactive/src

**Reply by somguyth** [reply]
Hi Mark,

Have the examples been added yet? I couldn't find them in the storybook, the Interactive section only had examples pertaining to zoom/pan/etc.
